user_sync/connector/directory_ldap.py

```diff
@@ -232,13 +232,15 @@ def find_group_dn(self, group):
                 group_dn = current_tuple[0]
         return group_dn
 
-    def iter_group_member_dns(self, group_dn, member_attribute, searched_dns = []):
+    def iter_group_member_dns(self, group_dn, member_attribute, searched_dns = None):
         """
         return group memberships dns from specified membership attribute in LDAP group object
         :type group: str
         :type member_attribute: str
         :rtype iterable(str)
         """
+        if searched_dns == None:
```

Comment:
Would recommend check to be changed to `if searched_dns is None`.  See [best practice](https://docs.quantifiedcode.com/python-anti-patterns/readability/comparison_to_none.html).